autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicOptionRow.tsx

```diff
@@ -16,7 +16,7 @@ interface BasicOptionRowProps {
  * option selected by this row, formatted as it is found in the Redux course cards
  */
 const BasicOptionRow: React.FC<BasicOptionRowProps> = ({ id, value }) => {
-  const option = useSelector<RootState, string>((state) => state.courseCards[id][value] || 'exclude');
+  const option = useSelector<RootState, string>((state) => state.courseCards[id][value] || 'no_preference');
```

Comment:
This changes the default for honors as well, can you instead make it so the default is `no_preference` when a card is created with `addCourseCard`?

Reply:
Oh yeah duh, this was the suggestion Adel gave. I was wondering why I hadn't done it previously when he suggested and this was the reason why.